### PR TITLE
Add mutations for tenant access create and delete

### PR DESCRIPTION
**Pull request by StanislavStefanov**
**Description**

Changes proposed in this pull request:
- Introduce addTenantAccess mutation
- Introduce removeTenantAccess mutation
 
**Related issue(s)**

- ...

**Pull Request status**

- [x] Implementation
- [x] Unit tests
- [x] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [x] Mocks are regenerated, using the automated script

### CODE REVIEW

**Comment by StanislavStefanov** [comment]
reset-vm

**Review by PetarTodorovv — CHANGES_REQUESTED** [review]

File: components/director/internal/repo/tenant_access_m2m.go [hunk 1]
```diff
@@ -47,7 +47,7 @@ const (
                     SELECT t2.id, t2.parent
                     FROM business_tenant_mappings t2
                              INNER JOIN parents t on t2.id = t.parent)
-			DELETE FROM %s WHERE %s AND owner = true AND tenant_id IN (SELECT id FROM parents)`
+			DELETE FROM %s WHERE %s AND tenant_id IN (SELECT id FROM parents)`
```
[the remark this answers is not in the document]
Comment (StanislavStefanov) [reply components/director/internal/repo/tenant_access_m2m.go:50]: This makes it imposible to delete records with access false. Also I have not faoud any reason to delete only the records with access true.

File: components/director/internal/domain/tenant/automock/business_tenant_mapping_converter.go
[the remark this answers is not in the document]
Comment (StanislavStefanov) [reply components/director/internal/domain/tenant/automock/business_tenant_mapping_converter.go:1]: regenerated them with the correct version

**Review by PetarTodorovv — APPROVED** [review]

**Review by nkanchev — APPROVED** [review]

**Pull request merged**